WidgetStick/widget-src/code.tsx

```diff
@@ -0,0 +1,44 @@
+const { editorType, widget } = figma;
+const { AutoLayout, Text, useStickable, useStickableHost, useSyncedState } =
+  widget;
+
+function Widget() {
+  const [stickable, setStickable] = useSyncedState("stickable", true);
+  // A widget can either be stickable or a stickable host at any given point in time, never both simultaneously.
+  // We are using state to toggle the behavior for the widget.
+  if (stickable) {
+    useStickable(({ newHostId, oldHostId }) =>
+      console.log({ newHostId, oldHostId })
```

Comment:
should we remove these log statements?

Reply:
This is subjective, but I find `console.log` indicating when the stickable callbacks are actually firing / what ids are present is fairly useful. I wanted to avoid a noop here, but I guess an alternative would be to write code that actually does something with the ids in those events?